app/actions/remote/channel.ts

```diff
@@ -544,6 +542,8 @@ export async function joinChannel(serverUrl: string, userId: string, teamId: str
         return {error};
     }
 
+    const userId = await getCurrentUserId(database);
```

Comment:
if we are always going to use the current user to join channels instead of using the `userId` we can then switch to use the value `me` instead?

Reply:
Maybe I am missing something, but that API doesn't seem to allow the `me` parameter.